packages/contracts/contracts/SemaphoreWhistleblowing.sol

```diff
@@ -8,7 +8,7 @@ import {SemaphoreGroups} from "@semaphore-protocol/contracts/base/SemaphoreGroup
 /// @title SemaphoreWhistleblowing
 /// @dev This contract uses the Semaphore base contracts to allow whistleblowers to leak information anonymously
 /// Leaks can be IPFS hashes, permanent links or other kinds of references.
-contract SemaphoreWhistleblowing is ISemaphoreWhistleblowing, SemaphoreGroups {
+abstract contract SemaphoreWhistleblowing is ISemaphoreWhistleblowing, SemaphoreGroups {
```

Comment:
It shouldn't be an abstract contract. And it shouldn't extend `SemaphoreGroups`.

Reply:
Hey I fixed this but havin some issue using the ```getmerkleTreeDuration```

Reply:
-> https://github.com/semaphore-protocol/extensions/pull/14#discussion_r1703835738